src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -58,6 +58,12 @@
    :auto-complete/next           {:binding "down"
                                   :fn      ui-handler/auto-complete-next}
 
+   :auto-complete/ctrl-prev      {:binding "ctrl+p"
```

Comment:
Looking at `:editor/redo`, looks like we can have multiple bindings with a vector. Any reason to create new entries instead of reusing prev and next?

Reply:
My fault, I thought that binding multiple is not supported before. Fixed it.
I found the shortcut of `ctrl+n/p` for auto-complete navigation is almost available from each app on Mac OS. Eg. Spotlight, Finder search, Chrome/Safari address bar, Github search bar. So I think it could be a default binding.
At the same time, I found that our shortcut key binding interface currently does not support binding multiple shortcut keys. I think this needs to be improved.